validate.py

```diff
@@ -87,6 +89,20 @@ def get_users():
         users.add(line.strip())
   return list(sorted(users))
 
+def last_commit_ts():
+  cmd = ['git', 'log', '-1', '--format=%ct']
```

Comment:
Do we want/need to explicitly specify the `master` branch? It would be safer to be explicit.

Reply:
added in 8489409